lib/transcryptor/decoder.rb

```diff
@@ -0,0 +1,63 @@
+# frozen_string_literal: true
+
+module Transcryptor
+  # Plugin module for VersionedFields gem
+  module Decoder
+    def decode_with_previous_settings(migration_data, **options)
+      # We only need to decode field value once
+      # Encoding will be performed by :attr_encypted itself
+      return public_send(migration_data.field) if @transcryptor_decode_migrated
+
+      field     = migration_data.field
+      current_v = migration_data.version
+
+      generate_versioned_fields!(field, current_v, options)
+
+      opts = encrypted_attributes[field]
+      public_send("#{field}_#{current_v}")
```

Comment:
Is this extra `public_send` necessary?

Reply:
No, I just forgot to remove this line. Updated.